client/extensions/woocommerce/app/settings/payments/payment-method-stripe.js

```diff
@@ -107,11 +117,25 @@ class PaymentMethodStripe extends Component {
 					{ this.renderEnabledField( method.settings.enabled.value ) }
 				</FormFieldset>
 				<FormFieldset className="payments__method-edit-field-container">
-					<FormLabel>{ translate( 'Enable Test Mode' ) }</FormLabel>
-					<PaymentMethodEditFormToggle
-						checked={ method.settings.testmode.value === 'yes' ? true : false }
-						name="testmode"
-						onChange={ this.onEditFieldHandler } />
+					<FormLabel>{ translate( 'Payment Mode' ) }</FormLabel>
+					<SegmentedControl
+						primary={ true }
```

Comment:
When setting attributes values to just true you can omit the value all together. So this can just be come:
`<SegmentedControl primary compact>`.